frontend/integration-tests/tests/login.scenario.ts

```diff
@@ -0,0 +1,74 @@
+import { $, browser, ExpectedConditions as until } from 'protractor';
+import { appHost } from '../protractor.conf';
+import * as loginView from '../views/login.view';
+import * as sidenavView from '../views/sidenav.view';
+import * as clusterSettingsView from '../views/cluster-settings.view';
+
+const JASMINE_DEFAULT_TIMEOUT_INTERVAL = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+const JASMINE_EXTENDED_TIMEOUT_INTERVAL = 1000 * 60 * 3;
+const {
+  CLUSTER_READER_IDP ='test',
+  CLUSTER_READER_USERNAME = 'test',
+  CLUSTER_READER_PASSWORD = 'test',
```

Comment:
This actually isn't a cluster-reader unless we're adding that cluster role binding somewhere. It's a normal user. I'd call these `HTPASSWD_IDP`, etc

Reply:
Ahh, okay